### ISSUE TITLE
Sudden drop in accuracy while training a deep neural net

### ISSUE THREAD

**Issue by saustar** [dmlc/mxnet#2041]
HI, I am using mxnet to train a 11-class image classifier. I am observing a weird behavior training accuracy was increasing slowly and went upto 39% and in next epoch it went down to 9% and then it stays close to 9% for rest of the training.
I restarted the training with saved model (with 39% training accuracy) keeping all other parameter same . Now training accuracy is increasing again. What can be the reason here ? I am not able to understand it . And its getting difficult to train the model this way as it requires me to see training accuracy values constantly.

**Comment by antinucleon** [comment]
It may because you are using too large learning rate.
On Thu, May 5, 2016 at 00:04 saustar <notifications@github.com> wrote:

> HI, I am using mxnet to train a 11-class image classifier. I am observing
> a weird behavior training accuracy was increasing slowly and went upto 39%
> and in next epoch it went down to 9% and then it stays close to 9% for rest
> of the training.
> I restarted the training with saved model (with 39% training accuracy)
> keeping all other parameter same . Now training accuracy is increasing
> again. What can be the reason here ? I am not able to understand it . And
> its getting difficult to train the model this way as it requires me to see
> training accuracy values constantly.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dmlc/mxnet/issues/2041>
>
-- 
Sent from mobile phone

**Comment by saustar** [comment]
I am using 0.01 learning rate but still If I restart the training it starts
increasing and right now accuracy become 42%.

On Thu, May 5, 2016 at 12:47 PM, Bing Xu <notifications@github.com> wrote:

> It may because you are using too large learning rate.
> On Thu, May 5, 2016 at 00:04 saustar <notifications@github.com> wrote:
>
> > HI, I am using mxnet to train a 11-class image classifier. I am observing
> > a weird behavior training accuracy was increasing slowly and went upto
> 39%
> > and in next epoch it went down to 9% and then it stays close to 9% for
> rest
> > of the training.
> > I restarted the training with saved model (with 39% training accuracy)
> > keeping all other parameter same . Now training accuracy is increasing
> > again. What can be the reason here ? I am not able to understand it . And
> > its getting difficult to train the model this way as it requires me to
> see
> > training accuracy values constantly.
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly or view it on GitHub
> > <https://github.com/dmlc/mxnet/issues/2041>
> >
> --
> Sent from mobile phone
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dmlc/mxnet/issues/2041#issuecomment-217091770>
>



-- 

Saurabh Gupta
+918377943044

**Comment by sxjscience** [comment]
@saustar Could you provide an example? In fact, I also meet with this kind of problem before but I failed to give an example...

**Comment by jiumem** [comment]
Try momentum with 0.5, or a smaller lr
